NGitLab.Tests/MergeRequest/MergeRequestClientTests.cs

```diff
@@ -50,10 +50,7 @@ public async Task Test_merge_request_api()
         var commits = mergeRequestClient.Commits(mergeRequest.Iid).All;
         Assert.That(commits.Any(), Is.True, "Can return the commits");
 
-        if (context.IsGitLabVersionInRange(VersionRange.Parse("[15.6,)"), out _))
-            Assert.That(mergeRequest.DetailedMergeStatus.EnumValue, Is.Not.Null);
-        else
-            Assert.That(mergeRequest.DetailedMergeStatus.EnumValue, Is.Null);
+        Assert.That(mergeRequest.DetailedMergeStatus.EnumValue, Is.Null);
```

Comment:
The previous code actually told us that, starting at version 15.6, a change in behavior was introduced. I would tend to keep it.